conf.yaml.example

```diff
@@ -20,3 +20,19 @@ BASIC_MODEL:
 #   base_url: https://ark-cn-beijing.bytedance.net/api/v3
 #   model: "doubao-1-5-thinking-pro-m-250428"
 #   api_key: xxxx
+
+# OTHER SETTINGS:
+# Search engine configuration
```

Comment:
It's the advanced setting of the Search engine. 
Currently it only supports tablet.